app/src/main/java/org/oppia/app/options/OptionsReadingTextSizeViewModel.kt

```diff
@@ -0,0 +1,18 @@
+package org.oppia.app.options
+
+import androidx.databinding.ObservableField
+
+/** ReadingText size settings view model for the recycler view in [OptionsFragment]. */
```

Comment:
Fix the sentence to
`/** ReadingTextSize settings view model for the recycler view in [OptionsFragment]. */`